examples/email-auth-local-storage/README.md

```diff
@@ -65,3 +67,30 @@ $ pnpm run dev
 ```
 
 This command will run a NextJS app on port 3000. If you navigate to http://localhost:3000 in your browser, you can follow the prompts to start an email auth.
+
+### 4/ Stackblitz Example
+
+You will need to create a Stackblitz account to run the following example
```

Comment:
I actually think you can run these without an account, which is amazing! Feel free to try in an incognito window